src/nnet3bin/nnet3-am-copy.cc

```diff
@@ -89,18 +89,17 @@ int main(int argc, char *argv[]) {
       ReadKaldiObject(set_raw_nnet, &nnet);
       am_nnet.SetNnet(nnet);
     }
-    
+
     if (learning_rate >= 0)
       SetLearningRate(learning_rate, &(am_nnet.GetNnet()));
 
-    if (scale != 0.0)
-      ScaleNnet(scale, &(am_nnet.GetNnet()));
+    ScaleNnet(scale, &(am_nnet.GetNnet()));
```

Comment:
Mostly for clarity, also for efficiency, would be good to have
if (scale != 1.0)
here.  Will merge anyway though.